Something/WebApplication/wwwroot/js/apps/authentication/components/AuthenticationForm/AuthenticationForm.tsx

```diff
@@ -0,0 +1,131 @@
+                                {
+                                    message: "Please enter a valid email address.",
+                                    required: true,
+                                },
+                            ],
+                        })(
+                            <Input
+                                className="ant-input-lg"
+                                placeholder="someone@example.com"
+                                name="Email"
+                            />,
+                        )}
+                    </FormItem>
+                    <b>Password</b>
+                    <FormItem>
+                        {getFieldDecorator("password", {
+                            rules: [
+                                {
+                                    message: "Password cannot be left blank.",
+                                    required: true,
+                                },
+                            ],
+                        })(
+                            <Input
+                                className="ant-input-lg"
+                                type="password"
+                                placeholder="********"
+                                name="Password"
+                            />,
+                        )}
+                    </FormItem>
+                    <FormItem>
+                        <Button
+                            type="primary"
+                            htmlType="submit"
+                            className="login-form-button ant-btn--rounded"
+                        >
+                            {domRepository.getButtonText()}
+                         </Button>
+                    </FormItem>
+                    {this.showErrorMessage(
+                        this.props.serverError,
+                        this.props.failedAttempt,
+                    )}
+                    <p className={`${container}__secondary-link`}>
+                        <a href={domRepository.getAlternateLink()}>
+                            {domRepository.getAlternateText()}
+                        </a>
+                    </p>
+                </Form>
+            </div>
+        );
+    }
+
+    private handleSubmit = (e: any) => {
+        e.preventDefault();
+        this.props.form.validateFields((err, values) => {
+            if (!err) {
+                e.currentTarget.submit();
+            }
+        });
+    }
+
+    private showErrorMessage = (serverError: boolean, failedAttempt: boolean): JSX.Element => {
+        let messageHeader: string;
+        let messageBody: string;
+        if (serverError) {
+            messageHeader = "Unexpected Error";
+            messageBody = "Whoops, something went wrong...";
+        } else if (failedAttempt) {
+            messageHeader = "Incorrect Email/Password";
+            messageBody = "Please try again or you can reset your password.";
+        } else {
+            return null;
```

Comment:
Invert this check.
`if (!serverError && !failedAttempt) return null`

Reply:
Done.